### PR TITLE
[FEATURE] Add Dzongkha

### PR DESCRIPTION
**Pull request by KinWang-2013**
This PR adds support for Dzongkha which is the national language of Bhutan 🇧🇹

### CODE REVIEW

**Comment by KinWang-2013** [comment]
@radar please check this

**Review by dblock — CHANGES_REQUESTED** [review]
Looks good! Can we add some more tests than just for 1 second, please?

I know it's unrelated, but we do need to fix CI. If you could please help out do the upgrades, we can probably drop 2.4 and add 3.2 and 3.1 (separate PR would be great).

**Comment by KinWang-2013** [comment]
> Looks good! Can we add some more tests than just for 1 second, please?
> 
> I know it's unrelated, but we do need to fix CI. If you could please help out do the upgrades, we can probably drop 2.4 and add 3.2 and 3.1 (separate PR would be great).

sure, I will

**Comment by KinWang-2013** [comment]
@dblock added some more tests, I will try doing upgrades in another PR

**Comment by KinWang-2013** [comment]
> @dblock added some more tests, I will try doing upgrades in another PR

@dblock please check, the other PR is #135

**Comment by dblock** [comment]
CI should pass now after #135, rebase?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 * [#131](https://github.com/radar/distance_of_time_in_words/pull/131): Deprecates `highest_measure_only`. Adds alternate form of `highest_measures` option to permit rounding up whatever part of the duration was previously silently discarded - [@seansfkelley](https://github.com/seansfkelley).
 * [#133](https://github.com/radar/distance_of_time_in_words/pull/133): Test on Ruby 3.0 and 3.1 - [@dblock](https://github.com/dblock).
+* [#134](https://github.com/radar/distance_of_time_in_words/pull/134): Add support for Dzongkha language (National language of Bhutan) - [@KinWang-2013](https://github.com/KinWang-2013)
```
Comment (dblock) [inline CHANGELOG.md:5]:
```suggestion
* [#134](https://github.com/radar/distance_of_time_in_words/pull/134): Add support for Dzongkha, the National language of Bhutan - [@KinWang-2013](https://github.com/KinWang-2013).
```

**Review by dblock — CHANGES_REQUESTED** [review]
Nit, missing a period in CHANGELOG, sorry!

**Comment by KinWang-2013** [comment]
> Nit, missing a period in CHANGELOG, sorry!

updated, thanks

**Review by dblock — APPROVED** [review]

**Pull request merged**